src/ImageSharp/Memory/DiscontiguousBuffers/MemoryGroup{T}.Consumed.cs

```diff
@@ -22,16 +23,31 @@ public Consumed(Memory<T>[] source, int bufferLength, long totalLength)
                 this.View = new MemoryGroupView<T>(this);
             }
 
-            public override int Count => this.source.Length;
+            public override int Count
+            {
+                [MethodImpl(InliningOptions.ShortMethod)]
```

Comment:
I've added that because those methods are being called directly from the enumerator: https://github.com/SixLabors/ImageSharp/blob/c237de595c40a251e9daa6f8a373ac45505755c0/src/ImageSharp/Memory/DiscontiguousBuffers/MemoryGroupEnumerator%7BT%7D.cs#L26

In theory (I hope), the JIT should be able to devirtualize and then inline that call, since all APIs are internal, so I assume it could potentially know the type is definitely just the one declared from there as a parameter. Of course, we could also declare those types as `sealed`, to be honest I'm not even sure why they're not already? 🤔

Reply:
I'm unsure about JIT here, anyways this is a question of low importance.

Reply:
Better one `AggressiveInlining` in excess than lacking one 🤣